doc/translation_status/translation_status_zh-CN.md

```diff
@@ -0,0 +1,197 @@
+# zh-CN translation status
+
+| EN file name | edit icon and badge |
+| --- | --- |
+| [README.md](../../README.md) | [📝](../../README_zh-CN.md) [![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3DREADME.md%26commit_hash%3Ddc7feca6ad1533e791b5fc1fd036636860f03d07)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=README.md&commit_hash=dc7feca6ad1533e791b5fc1fd036636860f03d07) |
+| [CODE_OF_CONDUCT.md](../../CODE_OF_CONDUCT.md) | [📝](../CODE_OF_CONDUCT/CODE_OF_CONDUCT_zh-CN.md) [![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3DCODE_OF_CONDUCT.md%26commit_hash%3Df2118ff45d9e46ca8fa44242363223be43b046dd)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=CODE_OF_CONDUCT.md&commit_hash=f2118ff45d9e46ca8fa44242363223be43b046dd) |
+| [CONTRIBUTING.md](../../CONTRIBUTING.md) | [📝](../CONTRIBUTING/CONTRIBUTING_zh-CN.md) [![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3DCONTRIBUTING.md%26commit_hash%3D00350f64a40b12f763a605bc16748d09379ab182)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=CONTRIBUTING.md&commit_hash=00350f64a40b12f763a605bc16748d09379ab182) |
+|  |  |
```

Comment:
What do you think that is the cause of this?

Reply:
The cells after this line are under the `doc/` directory as opposed to before this line, so this was added to delimit

Reply:
hmm
Personally, I think the blanks unnatural as a deliminator
It's possible that others will think it's a mistake, as I did
I think it is necessary to remove the blank or put in another distinct delimitation

Reply:
I see. How about delimitation is like this
```md
| ... | ... |
| doc/ |  |
| ... | ... |
```

Reply:
I'm sorry, but think it would be more consistent if it was removed
This is a matter of taste

btw, there is a file name change in the main branch, so this should fix the table link

Reply:
OK. They're links, so I did it